src/components/layout/BaseLayout.tsx

```diff
@@ -1,16 +1,20 @@
 import React, { FC, PropsWithChildren, ReactNode } from 'react';
 
-import { chakra, Container, Flex } from '@chakra-ui/react';
+import { chakra, Container } from '@chakra-ui/react';
 
 import { sizes } from 'src/themes/shared/sizes';
+import { breakpoints } from 'src/themes';
+import { useMediaQuery } from 'src/hooks';
 
+import Flex from '../flex';
 import Divider from '../divider';
 
 interface Props {
   header?: ReactNode;
   footer?: ReactNode;
   skyScraperAd?: ReactNode;
   maxContentWidth: keyof typeof sizes.container;
+  isSrpAds?: boolean;
```

Comment:
While I was working on the advanced search page ads, I noticed that there are also ad sizes with `500px`. Maybe it makes more sense to change the width pixels instead adding a boolean.  In case that we want to keep the boolean, I will adjust the props to something more generic. 

Thank you!